frontend/src/api/actions/positions.js

```diff
@@ -200,7 +200,11 @@ export const positionsSelector = createSelector(
                 ...rest
             }) => ({
                 ...rest,
-                instructors: instructor_ids.map((x) => instructorsById[x]),
+                // When the instructor list references an instructor that we haven't loaded
+                // we don't want the frontend to crash, so filter out any null instructors
+                instructors: instructor_ids
+                    .map((x) => instructorsById[x])
+                    .filter((x) => x),
```

Comment:
Is this really the same as `x !== null`?
Cool...

Reply:
It's not the same. It's the same as `!!x === true`